js-arcade/tictactoe/TicTacToe.js

```diff
@@ -16,7 +19,55 @@ function TicTacToe() {
             this.xIsNext = !this.xIsNext;
         }
     };
-    this.isGameOver = function(x, y) {
+    this.isGameOver = function (x, y) {
+        let current_player = this.xIsNext ? true : false;
```

Comment:
xIsNext is already a boolean, so you could just say "let current_player = this.xIsNext" (tho you can of course just reference this.xIsNext directly, since you never assign it - only read it - but I get if you just want a local variable with a different name)

Reply:
You could also use const here